install/kubernetes/cilium/templates/hubble-relay/deployment.yaml

```diff
@@ -71,11 +71,9 @@ spec:
               protocol: TCP
             {{- end }}
           readinessProbe:
-            tcpSocket:
-              port: grpc
+            {{- include "hubble-relay.probe" . | nindent 12 }}
```

Comment:
I know the old code didn't do it, but we should strongly consider adding `initialDelaySeconds` and/or a `startupProbe`, especially now where the probe indicates actual readiness and might take a few seconds to connect to the peer service and the first peer.

It might also be worth setting `timeoutSeconds: 3`, that's what other ciilum components (e.g. agent, operator) do.

https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-startup-probes